docs/pages/server-access/guides/gcp-discovery.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: Automatically Discover GCP Compute Instances (Preview)
```

Comment:
We're revising our Preview policy and no longer going to call every single thing we release as preview. Let's remove this.
```suggestion
title: Automatically Discover GCP Compute Instances
```